android/build.gradle

```diff
@@ -3,9 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.+'
```

Comment:
That's pretty cool.
How confindent are we in making this change vs where the main RN project is?
Will this introduce any unwanted (or different) behavior?